<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>318</width>
      <height>143</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleBottomValue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>318</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-9999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleTopValue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>318</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-9999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMinus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDivide">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTimes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSqrt">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sqrt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
